docs/collection-tables.rst

```diff
@@ -0,0 +1,89 @@
+.. _collection-tables:
+
+=================
+Collection Tables
```

Comment:
Uninformative title. I would not find this functionality if I didn't know where to look. I can't think of a better alternative either.

Reply:
I am against (an)other title, since inventing another term just for the purpuse of title will only increase confusion. If we do decide for a more meaningful rename, we should also rename `CollectionTable` class name.